packages/client-db/src/Sync/worker.js

```diff
@@ -0,0 +1,158 @@
+// Copyright 2017-2018 @polkadot/client-db authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+// import { Message } from './types';
+
+// import { parentPort } from 'worker_threads';
+const { parentPort } = require('worker_threads');
+
+// import Trie from '@polkadot/trie-db';
+const Trie = require('@polkadot/trie-db').default;
+
+// import commands from './commands';
+// Dangit, also doesn't reolve .ts here... erk :(
+// const commands = require('./commands').default;
+const commands = {
+  START: 0x00,
+  SIZE: 0x01,
+  FILL: 0x02,
+  READ: 0x03,
+  END: 0x0f,
+  ERROR: 0xff
+};
```

Comment:
FYI, I'm doing something similar in dotstats, found this helpful for statically checking stuff:
https://github.com/polkadot-js/dotstats/blob/81ef3ee14e49a5744279cdc59900cb37f74cc475/packages/common/src/feed.ts#L4-L12

`0 as 0` ensures the type is `0` instead of `number`.